qiskit/circuit/quantumcircuit.py

```diff
@@ -4683,6 +4685,202 @@ def mcp(
             copy=False,
         )
 
+    def mcrx(
+        self,
+        theta: ParameterValueType,
+        q_controls: Union[QuantumRegister, List[Qubit]],
+        q_target: Qubit,
+        use_basis_gates: bool = False,
+    ):
+        """
+        Apply Multiple-Controlled X rotation gate
+
+        Args:
+            self (QuantumCircuit): The QuantumCircuit object to apply the mcrx gate on.
+            theta (float): angle theta
+            q_controls (QuantumRegister or list(Qubit)): The list of control qubits
+            q_target (Qubit): The target qubit
+            use_basis_gates (bool): use p, u, cx
```

Comment:
`self` is implicit, we don't need to list it 🙂 

Also we typically don't duplicate the type hints here, if they are already in the signature we can skip them in the docstring:
```suggestion
            theta: angle theta
            q_controls: The list of control qubits
            q_target: The target qubit
            use_basis_gates: use p, u, cx
```